src/main/java/UI.java

```diff
@@ -0,0 +1,61 @@
+import java.util.List;
+
+/**
+ * Manages UI related methods.
+ */
+public class UI {
+
+    public UI() {
+    }
+
+    public String horizontalLine() {
+        return "__________________________________________________";
+    }
+
+    /**
+     * Greetings when the program starts
+     */
+    public String greet() {
+        /*horizontalLine();
+        System.out.println("Hello I'm Duke");
+        System.out.println("What can I do for you?");
+        horizontalLine();
+        System.out.println();*/
```

Comment:
It might be better to remove this comment part since you can always refer back to git history if you need to compare your code.

Reply:
yup, thanks!